datasets/enriched_web_nlg/enriched_web_nlg.py

```diff
@@ -0,0 +1,189 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""The Enriched WebNLG corpus"""
+
+from __future__ import absolute_import, division, print_function
+
+import os
+import xml.etree.cElementTree as ET
+from collections import defaultdict
+from glob import glob
+from os.path import join as pjoin
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{ferreiraetal2018,
+  author = 	"Castro Ferreira, Thiago and Moussallem, Diego and Wubben, Sander and Krahmer, Emiel",
+  title = 	"Enriching the WebNLG corpus",
+  booktitle = 	"Proceedings of the 11th International Conference on Natural Language Generation",
+  year = 	"2018",
+  series = {INLG'18},
+  publisher = 	"Association for Computational Linguistics",
+  address = 	"Tilburg, The Netherlands",
+}
+"""
+
+_DESCRIPTION = """\
+WebNLG is a valuable resource and benchmark for the Natural Language Generation (NLG) community. However, as other NLG benchmarks, it only consists of a collection of parallel raw representations and their corresponding textual realizations. This work aimed to provide intermediate representations of the data for the development and evaluation of popular tasks in the NLG pipeline architecture (Reiter and Dale, 2000), such as Discourse Ordering, Lexicalization, Aggregation and Referring Expression Generation.
+"""
+
+_HOMEPAGE = "https://github.com/ThiagoCF05/webnlg"
+
+_LICENSE = "CC Attribution-Noncommercial-Share Alike 4.0 International"
+
+_URL = "https://github.com/ThiagoCF05/webnlg/zipball/master/"
```

Comment:
You need to set the commit sha in the url, otherwise the data may be not deterministic